packages/connectors/src/Integrations/BankOfCanada/.claspignore

```diff
@@ -2,9 +2,8 @@
 !Integrations/BankOfCanada/*.js
 !appsscript.json
 !Core/Abstract*.js
-!Core/*Storage.js
-!Core/GoogleSheets*.js
```

Comment:
@zapolsky please check if GoogleSheetsConfig will be pushed with` clasp push ` if !Core/GoogleSheets*.js is removed from .claspignore